content/desktop/settings/windows.md

```diff
@@ -216,6 +212,22 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
 If you are running Windows containers in Docker, you can allow the Windows Docker daemon to use Docker Desktop's internal proxy, with the **Use proxy for Windows Docker daemon** setting.
 This is useful when a corporate proxy that requires authentication is manually configured or set at the system level. If you are an admin for your organization and have a Docker Business subscription, you can control this setting with [Settings management](../hardened-desktop/settings-management/configure.md) using the `windowsDockerdPort` parameter.
 
+#### Proxy authentication
+
+Docker Desktop supports Basic, Kerberos and NTLM proxy authentication methods. 
+
+##### Basic authentication
+
+If your proxy uses Basic authentication, Docker Desktop prompts developers for a username and password and caches the credentials. All passwords are stored securely in the OS credential store. It will request re-authentication if that cache is removed.
+
+It's recommended that you use an `https://` URL for HTTP/HTTPS proxies to protect passwords during network transit. Docker Desktop also supports TLS 1.3 for communication with proxies.
+
+##### Kerberos and NTLM authentication
+
+Kerberos and NTLM proxy authentication are available for Pro, Team, and Business subscribers with Docker Desktop version 4.30 and later. No additional configuration is needed beyond specifying the proxy IP address and port.
```

Comment:
@aevesdocker  as per the discussion here: https://docker.slack.com/archives/C0W4XJVFW/p1714486276644979?thread_ts=1714127067.533579&cid=C0W4XJVFW

there has been a change to the required subscription level for Kerberos and NTLM authentication. Can we update it to:
"Kerberos and NTLM proxy authentication are available for Business subscribers with Docker Desktop version 4.30 and later."